documentation/docs/UI/channel_list/binding_view_models.md

```diff
@@ -0,0 +1,29 @@
+---
+id: uiChannelListBindingViewModels
+title: Binding View Models
+sidebar_position: 2
+---
+The Android SDK comes together with view models for view components which are responsible for providing all necessary data. You need to do two things to connect a particular view with its view model:
+```kotlin
+// Step 1: Create ViewModels
+val channelListHeaderViewModel: ChannelListHeaderViewModel by viewModels()
+val channelListFactory: ChannelListViewModelFactory = ChannelListViewModelFactory(
+    filter = Filters.and(
+        Filters.eq(“type”, “messaging”),
+        Filters.`in`(“members”, listOf(ChatDomain.instance().currentUser.id)),
+    ),
+    sort = QuerySort.desc(Channel::lastUpdated),
+    limit = 30
+)
+val channelListViewModel: ChannelListViewModel by viewModels { channelListFactory }
+// Step 2: Bind views with view models
+channelListHeaderViewModel.bindView(channelListHeaderView, viewLifecycleOwner)
+channelListViewModel.bindView(channelListView, viewLifecycleOwner)
+```
+This is how default channel list should look like:
+
+| Light Mode | Dark Mode |
+| --- | --- |
+|![light mode](https://user-images.githubusercontent.com/17440581/108333708-877aaf00-71d1-11eb-9998-1a68e1b8fe63.png)|![dark mode](https://user-images.githubusercontent.com/17440581/108333703-85b0eb80-71d1-11eb-8fb0-551dd5065587.png)|
```

Comment:
We shouldn't do that. Let's place images inside `static/images` Here are the docs on how to do that: https://docusaurus.io/docs/static-assets